virttest/asset.py

```diff
@@ -494,9 +494,13 @@ def download_file(asset_info, interactive=False, force=False):
         else:
             answer = 'y'
         if answer == 'y':
-            download.url_download_interactive(url, destination,
-                                              "Downloading %s" % title)
-            had_to_download = True
+            try:
+                download.url_download_interactive(url, destination,
+                                                  "Downloading %s" % title)
+                had_to_download = True
+            except Exception, download_failure:
+                logging.error("Check your internet connection: %s",
```

Comment:
can it proceed if the download of the file fails, if cant proceed then it is better to raise the error instead of logging.